src/lib.rs

```diff
@@ -126,30 +126,32 @@ impl<L: LogLevel> Verbosity<L> {
         self.log_level().is_none()
     }
 
-    fn verbosity(&self) -> i8 {
-        level_value(L::default()) - (self.quiet as i8) + (self.verbose as i8)
+    fn verbosity(&self) -> u8 {
```

Comment:
Looks like this PR is blocked on #115

Reply:
Yep, I split off the changes to make the u8 parts of the change easier to evaluate on their own. Github doesn't have a good way to do stacked PRs unfortunately.

Reply:
I know, I deal with that a lot.  This is more of a bookkeeping comment to make sure we don't move forward without that being merged first.  When contributing, I recommend making these relationships explicit, rather than relying on maintainers to catch them.

Reply:
At this point, this PR needs a rebase and this item can be resolved.